azure-functions-sdk/src/templates/new/activity.rs.hbs

```diff
@@ -0,0 +1,6 @@
+use azure_functions::{bindings::DurableActivityContext, func};
+
+#[func]
+pub async fn {{name}}(context: DurableActivityContext) {
+    context.set_output("Hello from {{name}}!");
+}
```

Comment:
Nit: I think the other templates have a trailing newline.  Let's add one here too.